projects/canopy/src/lib/promo-card/promo-card.notes.ts

```diff
@@ -0,0 +1,135 @@
+export const notes = `
+# Promo Card List Component
+
+## Purpose
+
+Promo Cards allow you to arrange content in a card-like manner for promotional purposes. Promo Cards are organised by a
+Promo Card List and will be displayed in columns on desktops and stacked on mobile devices.
+
+### Content guidelines
+
+Each Promo Card should respect the following conventions:
+
+- Title - max 33 characters (with spaces)
```

Comment:
I wondered wether we should mention the max character count here? I don't dislike it, but we haven't done it elsewhere. This would maybe be more of a design guideline than a technical one.

Reply:
Good question - we originally came up with this because Seb asked if we could enforce it in the code. We mentioned it's not a great idea to go down that route and instead rely on everyone following the guidelines. So talked about recording in it both designer and developer docs to spread the message wide. My gut feel is that it's okay to have it to keep in the dev docs, because Promo Cards are kind of an outlier - they have these quite constrictive guidelines - whereas other components are much more freeform. Happy to remove though, if it's the consensus 👍